packages/host/app/components/operator-mode/overlays.gts

```diff
@@ -37,6 +37,10 @@ export default class OperatorModeOverlays extends Component<Signature> {
     );
   }
 
+  isPillDisplay(renderedCard: RenderedCardForOverlayActions) {
```

Comment:
Probably we can find a more descriptive name. Something that tells that this is a collection so it's clearer in the template why we don't want the overlay header